PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs

```diff
@@ -17,6 +18,18 @@ public static class CatchNearbyPokemonsTask
     {
         public static void Execute(Context ctx, StateMachine machine)
         {
+            var invResp = ctx.Client.Inventory.GetInventory();
+            var invRespRes = invResp.Result;
+            if (invRespRes.Success)
+            {
+                var items = invResp.Result.InventoryDelta.InventoryItems;
+                if (items.Select(rawItem => rawItem.InventoryItemData.Item).Any(item => (item.ItemId == ItemId.ItemPokeBall || item.ItemId == ItemId.ItemGreatBall || item.ItemId == ItemId.ItemUltraBall || item.ItemId == ItemId.ItemMasterBall) && item.Count == 0))
+                {
+                    Logger.Write("No pokeballs, ignoring pokemon encounters.", LogLevel.Warning);
```

Comment:
We don't use the Logger from the Logic project anymore, create an event, or fire a WarnEvent